### PR TITLE
fix typos in readme

### PR DESCRIPTION
**Pull request by elseym**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ Requires
...
-* in associates arrays, the "=>" operators must be aligned
-* in arrays, the key and '=>' operator must be on the same line
-* each consecutive variable assignments must align on the assignment operator
-* use statements must be sorted
-* you should use the imported class name, whenever it was imported with a use statement
-* variables in double quoted strings must be surrounded by { }, e.g. "{$VAR}" instead of "$VAR"
-* sprintf or "{$VAR1} {$VAR2}" must be used instead of the concat operator, concat operators are only allowed to concat constants and multi line strings,
-* a white space is required after each cast operator, e.g. "(int) $value" instead of "(int)$value",
+* short array syntax `[...]` must be used instead of  `array(...)`
+* in assiciative arrays, the `=>` operators must be aligned
```
Comment (mmoll) [inline README.md]: associative

Comment (elseym) [inline README.md]: you, sir, are my most favourite human linter.

File: README.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ Requires
...
+* in arrays, the key and `=>` operator must be on the same line
+* each consecutive variable assignment must align at the assignment operator
+* use statements must be sorted lexicographically
+* you should use the imported class name when it was imported with a use statement
+* interpolated variables in doubly quoted strings must be surrounded by `{ }`, e.g. `{$VAR}` instead of `$VAR`
```
Comment (mmoll) [inline README.md]: double

File: README.md [hunk 3]
```diff
@@ -13,15 +13,15 @@ Requires
...
+* `sprintf` or `"{$VAR1} {$VAR2}"` must be used instead of the dot operator; concat operators are only allowed to concatenate constants and multi line strings,
+* a white space is required after each typecast, e.g. `(int) $value` instead of `(int)$value`
```
Comment (mmoll) [inline README.md]: not brought in by this PR, but it's "whitespace"

File: README.md (hunk 2 above)
Comment (elseym) [inline README.md]: ...'doubly quoted string' as in 'doubly linked list'?

File: README.md (hunk 3 above)
Comment (elseym) [inline README.md]: will update that, too

File: README.md (hunk 2 above)
Comment (elseym) [inline README.md]: https://secure.php.net/manual/en/language.types.string.php#language.types.string.syntax.double

that should settle it :)

Comment (mmoll) [inline README.md]: then it would be "doubly-quoted"

**Comment by mmoll** [comment]
dankeschön @elseym!

**Pull request merged**